### PR TITLE
Vignette

### PR DESCRIPTION
**Pull request by natesheehan**
Adding this PR here - but this article could still do with some more work. I haven't included any of the plots in here, but I have left the comments which allow the user to test themself - whats your thoughts on this @Robinlovelace ?

### CODE REVIEW

File: vignette/montlake_seattle.Rmd [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Introduction 
+Lets start with the city A/B street began, Seattle. U.S. The seaport city is home to over 700,000 people, including A/B Street creator, Dustin Carlino, who has been developing tools to empirically study the impact of small changes within the road network: this means "you can transform that street parking into a bus lane or fix that pesky left turn at a traffic signal, measure the effects, then propose actually making the change". For the past two years, Seattle has been a key are of study within the A/B street simuverse and thus makes it a great starting point in understanding the utilities of `abstr` to generate site data for A/B street.
```
Comment (Robinlovelace) [inline vignette/montlake_seattle.Rmd:20]: This is great, liking the intro.

File: vignette/montlake_seattle.Rmd [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+<!-- ## Install packages -->
+<!-- ```{r, eval = FALSE} -->
+<!-- devtools::install_cran(tidyverse) -->
+<!-- ``` -->
+
+## Example
+This example demonstrates how to wrangle the data for the three key components needsed to generate scenarios for A/B street. These components are: OD data, site zones and site buildings. With these components, the `abstr` package is ready to convert dataframes into simulations! Lets get started.
+
+### Load packages
+```{r, eval = FALSE}
```
Comment (Robinlovelace) [inline vignette/montlake_seattle.Rmd:31]: Good solution to the pkg dependency conundrum. Tidyverse is too easy to use and popular not to recommend...

File: vignette/montlake_seattle.html [hunk 3]
```diff
@@ -0,0 +1,350 @@
+<!DOCTYPE html>
```
Comment (Robinlovelace) [inline vignette/montlake_seattle.html:1]: I think this file should be deleted. Just the .Rmd is enough for pkgdown to generate the html.

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Looking good. I think this is good on showing people how to start with the raw data and moving through the whole process to get to the end product: an A/B Street simulation represented by a GIF. Long term this could move into a 'data cleaning to generate the Montlake data' vignette but we can worry about moving parts of it around later. For now this is just what's needed for people getting started, just need to purge the .html file and it's good to go I reckon.

File: vignette/montlake_seattle.Rmd (hunk 2 above)
Comment (natesheehan) [reply vignette/montlake_seattle.Rmd:31]: I believe we had a similar issue with PCT - should we also generate an rpubs with plots?

**Comment by natesheehan** [comment]
> 'data cleaning to generate the Montlake data' vignette 

Yeah, that sounds like a great plan for the next version of this! I'll do the reproduction for Leeds next (and maybe Exeter at the same time).

File: vignette/montlake_seattle.Rmd (hunk 2 above)
Comment (Robinlovelace) [reply vignette/montlake_seattle.Rmd:31]: > I believe we had a similar issue with PCT - should we also generate an rpubs with plots?

It's a balance between reproducibility and package dependencies and run times. There are good balances to be found most of the time but when you want to process a load of data using `eval=FALSE` and linking to an externally hosted file that shows the whole thing like RPubs or even a GitHub issue or gist is a good shout. 

Partial solution to the `tidyverse` conundrum, compromise and Suggest `dplyr`:

https://github.com/a-b-street/abstr/blob/a0d2507b1157eeceda1f5f7819703e55867016fd/DESCRIPTION#L41-L42

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**